src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -120,12 +120,13 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     network = dataprocConfig.vpcNetwork.map(VPCNetworkName),
     targetTags = List(NetworkTag(dataprocConfig.networkTag)))
 
-  // Startup script to install on the cluster master node. This allows Jupyter to start back up after
-  // a cluster is resumed.
+  // Startup scripts to install on the cluster master node.
+  // This allows Jupyter and Welder to start back up after a cluster is resumed.
   //
   // The || clause is included because older clusters may not have the run-jupyter.sh script installed,
   // so we need to fall back running `jupyter notebook` directly. See https://github.com/DataBiosphere/leonardo/issues/481.
   private lazy val masterInstanceStartupScript: immutable.Map[String, String] = {
+ //TODO
```

Comment:
Important TODO!  Until this is done, restarting the cluster after stopping it will NOT restart Welder.

Reply:
wouldn't it just be something like adding `docker exec -d ${welderServerName} /bin/bash -c '/opt/bin/../server'

Reply:
yeah I don't think this will be difficult

Reply:
Confirmed: starting and stopping a cluster cause Jupyter to restart correctly but not Welder.

After doing this, ssh-ing into the Cluster Master Node and executing `docker exec -d -u daemon welder-server /opt/docker/bin/server start` started Welder again.

Reply:
This works now.